src/components/modules/chat/ui/components/ChatInput.tsx

```
@@ -0,0 +1,154 @@
+  const {
+    attachedFiles,
+    isUploading,
+    dragActive,
+    hasFiles,
+    handleFileSelect,
+    handleDrop,
+    handleDragOver,
+    handleDragLeave,
+    removeFile,
+    clearFiles,
+    openFilePicker,
```

ESLint

[error] 'openFilePicker' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)